vochain/state/snapshot.go

```diff
@@ -447,7 +447,7 @@ type DBPair struct {
 func ExportNoStateDB(w io.Writer, reader *NoState) (uint32, error) {
 	pairs := []DBPair{}
 	err := reader.Iterate(nil, func(key []byte, value []byte) bool {
-		pairs = append(pairs, DBPair{Key: key, Value: value})
```

Comment:
i came across this bug while working on #1235 , fixed it there but i also checked other uses of db.Iterate and so i made this into a separate PR